logp/global_test.go

```diff
@@ -80,7 +80,8 @@ func TestRecover(t *testing.T) {
 	const recoveryExplanation = "Something went wrong"
 	const cause = "unexpected condition"
 
-	DevelopmentSetup(ToObserverOutput())
+	err := DevelopmentSetup(ToObserverOutput())
+	assert.NoError(t, err)
```

Comment:
Same thing here (`require` instead of `assert`), does it make sense to continue the test if the development setup fails?
```suggestion
	require.NoError(t, err)
```